### DISCUSSION TITLE
Evaluate an alternative to Lerna

### DISCUSSION THREAD

**Discussion by silentHoo** [inovex/elements#450] [Ideas]
TBD

**Comment by pfecht** [comment]
@janivo sth else on the list that I just saw in stencil slack:
https://github.com/changesets/changesets

Not sure how this helps, but might be nice with focus on changelog gen.

**Reply by janivo** [reply]
Yea, I've seen that as well as some other interesting tools [in this collection of turborepo](https://turborepo.org/docs/guides/complimentary-tools). Might be worth a look
